packages/material-ui/src/Tabs/Tabs.test.js

```diff
@@ -96,34 +113,34 @@ describe('<Tabs />', () => {
 
   describe('prop: className', () => {
     it('should render with the user and root classes', () => {
-      const wrapper = shallow(
+      const wrapper = mount(
         <Tabs width="md" onChange={noop} value={0} className="woofTabs">
           <Tab />
           <Tab />
         </Tabs>,
       );
-      assert.strictEqual(wrapper.hasClass('woofTabs'), true);
-      assert.strictEqual(wrapper.hasClass(classes.root), true);
+      const root = findOutermostIntrinsic(wrapper);
+      assert.strictEqual(root.hasClass('woofTabs'), true);
+      assert.strictEqual(root.hasClass(classes.root), true);
     });
   });
 
   describe('prop: centered', () => {
     it('should render with the centered class', () => {
-      const wrapper = shallow(
+      const wrapper = mount(
         <Tabs width="md" onChange={noop} value={0} centered>
           <Tab />
           <Tab />
         </Tabs>,
       );
       const selector = `.${classes.flexContainer}.${classes.centered}`;
-      assert.strictEqual(wrapper.find(selector).name(), 'div');
-      assert.strictEqual(wrapper.find(selector).length, 1);
+      assert.strictEqual(wrapper.find(selector).type(), 'div');
     });
   });
 
   describe('prop: children', () => {
-    it('should accept an invalid child', () => {
```

Comment:
Don't use these generic terms when describing a test. Be explicit about what "invalid" means. Of course it warns about "invalid" values. What we're interested in is what invalid means for the unit we're testing.

Reply:
I don't see how using a descriptive description helps. It's duplicating the information already available by reading the test case.